packages/wdio-allure-reporter/src/index.js

```diff
@@ -199,13 +199,15 @@ class AllureReporter extends WDIOReporter {
             return
         }
 
-        // add hook as test to suite
-        this.onTestStart(hook)
+        // add hook as test to suite if not mocha All hook
+        if (!isMochaAllHooks(hook.title)) {
```

Comment:
To be consistent with other if's
```
if (isMochaAllHooks(hook.title)) {
  return
}

this.onTestStart(hook);
```

Reply:
BTW do we even need to add hook as test to suite on the last line? We ignore global hooks, have handles for each and all hooks. Which hooks are left?

Reply:
> To be consistent with other if's
> 
> ```
> if (isMochaAllHooks(hook.title)) {
>   return
> }
> 
> this.onTestStart(hook);
> ```

BTW do we even need to add hook as test to suite on the last line? We ignore global hooks, have handles for each and all hooks. Which hooks are left?

Reply:
e.g. some cucumber hooks. I am not sure how reporter handles them